GG/GG/Wnd.h

```diff
@@ -194,6 +194,30 @@ extern GG_API const WndFlag NO_WND_FLAGS;
     current.  Note the use of the phrase "client-area children".  This refers
     to children entirely within the client area of the window.
 
+    <h3>Frame Timing</h3>
+
+    <br>Each frame is a cycle culminating in rendering the screen.  The
+    following things happen, in order:
+      + events from the GUI and the data sources are handled
+      + PreRender() is called to update layout if required.
+      + Render() is called.
+
+    Wnd does not distinguish between events from the GUI and events from the
+    data sources.
+
+    PreRender() and Render() are only called if the window is visible.
+
+    PreRender() defers all layout changes until after all events have been
```

Comment:
this is a bit confusing... does PreRender do things inherently, or should various derived classes set themselves up so that their Layout changes in response to (eg.) SizeMove events are only done in PreRender?